online-shop/src/API/CartProducts.tsx

```diff
@@ -0,0 +1,15 @@
+// Initially, the cart is empty
+
+const ProductsInCart = {
+    products: [],
+    all: function() { return this.products},
+    // We will use this function from the Product Detail Page
+    add: function(product) {
+        this.products.push(product)
+    },
+    // Get only the ids and quantity for making an order
+    id: function () { return this.products.map(function(product) {
```

Comment:
Same here, rename it to something like _getCartProductIds_ or better even _getCartProductDetails_, since you are also returning the quantity. By the way, why is quantity hard-coded as `1`, instead of returning the actual quantity?